### ISSUE TITLE
Mixed benchmark doesn't test what they should test

### ISSUE THREAD

**Issue by Zelldon** [camunda/zeebe#11845]
**Describe the bug**

We created the mixed benchmarks to run different processes with different elements, etc. It looks like we lost them on the way the most recent benchmarks (currently all of them) are just running with the normal starter.

If we take a look at the medic benchmark overview, someone could think everything is fine since all running the same load. But instead of 25 we should reach 100.

![mixed](https://user-images.githubusercontent.com/2758593/221792802-115be610-fc09-4615-a0ab-378e78daebf9.png)

When we check the Kubernetes namespace:
 *  we see that the publisher is not running.
 * We see timer **is** running (but in the metrics we see no catch events)

```
 
$ kgpo
NAME                                                              READY   STATUS      RESTARTS   AGE
camunda-platform-curator-27959520-xgmct                           0/1     Completed   0          13m
elasticsearch-master-0                                            1/1     Running     0          10h
elasticsearch-master-1                                            1/1     Running     0          10h
elasticsearch-master-2                                            1/1     Running     0          10h
leader-balancer-27959520-t4g46                                    0/1     Completed   0          13m
medic-y-2023-cw-09-e5f0b66-benchmark-mixed-zeebe-0                1/1     Running     0          9h
medic-y-2023-cw-09-e5f0b66-benchmark-mixed-zeebe-1                1/1     Running     0          10h
medic-y-2023-cw-09-e5f0b66-benchmark-mixed-zeebe-2                1/1     Running     0          10h
medic-y-2023-cw-09-e5f0b66-benchmark-mixed-zeebe-gateway-chqxxv   1/1     Running     0          10h
medic-y-2023-cw-09-e5f0b66-benchmark-mixed-zeebe-gateway-cxh8wh   1/1     Running     0          10h
starter-68646b5bfb-8bgd8                                          1/1     Running     0          9h
timer-84c54f57d4-lhlmx                                            1/1     Running     0          9h
worker-7bd97bd566-cvfh8                                           1/1     Running     0          10h
worker-7bd97bd566-p2htq                                           1/1     Running     0          9h
worker-7bd97bd566-w6t8n                                           1/1     Running     0          9h
```

![nocatch](https://user-images.githubusercontent.com/2758593/221793472-5ae43091-39c5-46f5-94c6-21496430133d.png)

We see in the current events that only normal process instances are created.

**To Reproduce**

Run a mixed benchmark

**Expected behavior**

We run our expected workload on the mixed benchmarks.

**Comment by Zelldon** [comment]
One issue was a typo in deploying the mixed benchmarks, which causes that publishers are not deployed #11861 

Another seems to be that the configuration is not correctly applied to the application


Starter log, which prints at start the config:

```
[cqjawa zeebe/ cluster: zeebe-cluster ns:zell-mixed-investigation]$ k logs starter-c8f86b765-d8k2j 
NOTE: Picked up JDK_JAVA_OPTIONS: -Dconfig.override_with_env_vars=true -Dapp.brokerUrl=zell-test-zeebe-gateway:26500 -Dapp.starter.rate=150 -Dapp.starter.durationLimit=0 -Dzeebe.client.requestTimeout=62000 -XX:+HeapDumpOnOutOfMemoryError
08:39:44.101 [main] INFO  io.camunda.zeebe.App - Starting app with config: {
    # system properties
    "brokerUrl" : "zell-test-zeebe-gateway:26500",
    # application.conf @ file:/app/resources/application.conf: 5
    "monitoringPort" : 9600,
    # merge of system properties,application.conf @ file:/app/resources/application.conf: 7
    "starter" : {
        # application.conf @ file:/app/resources/application.conf: 11
        "bpmnXmlPath" : "bpmn/one_task.bpmn",
        # system properties
        "durationLimit" : "0",
        # application.conf @ file:/app/resources/application.conf: 16
        "msgName" : "msg",
        # application.conf @ file:/app/resources/application.conf: 12
        "payloadPath" : "bpmn/big_payload.json",
        # application.conf @ file:/app/resources/application.conf: 8
        "processId" : "benchmark",
        # system properties
        "rate" : "150",
        # application.conf @ file:/app/resources/application.conf: 17
        "startViaMessage" : false,
        # application.conf @ file:/app/resources/application.conf: 10
        "threads" : 2,
        # application.conf @ file:/app/resources/application.conf: 13
        "withResults" : false,
        # application.conf @ file:/app/resources/application.conf: 14
        "withResultsTimeout" : "60s"
    },
    # application.conf @ file:/app/resources/application.conf: 4
    "tls" : false,
    # application.conf @ file:/app/resources/application.conf: 20
    "worker" : {
        # application.conf @ file:/app/resources/application.conf: 24
        "capacity" : 30,
        # application.conf @ file:/app/resources/application.conf: 27
        "completeJobsAsync" : false,
        # application.conf @ file:/app/resources/application.conf: 26
        "completionDelay" : "300ms",
        # application.conf @ file:/app/resources/application.conf: 21
        "jobType" : "benchmark-task",
        # application.conf @ file:/app/resources/application.conf: 28
        "payloadPath" : "bpmn/big_payload.json",
        # application.conf @ file:/app/resources/application.conf: 25
        "pollingDelay" : "1s",
        # application.conf @ file:/app/resources/application.conf: 23
        "threads" : 10,
        # application.conf @ file:/app/resources/application.conf: 22
        "workerName" : "benchmark-worker"
    }
}

08:39:45.500 [main] DEBUG io.netty.util.internal.logging.InternalLoggerFactory - Using SLF4J as the default logging framework
08:39:45.503 [main] DEBUG io.netty.util.internal.PlatformDependent0 - -Dio.netty.noUnsafe: false
08:39:45.504 [main] DEBUG io.netty.util.internal.PlatformDependent0 - Java version: 17
08:39:45.505 [main] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.theUnsafe: available
08:39:45.505 [main] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.copyMemory: available
08:39:45.505 [main] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.storeFence: available
08:39:45.506 [main] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.Buffer.address: available
08:39:45.506 [main] DEBUG io.netty.util.internal.PlatformDependent0 - direct buffer constructor: unavailable: Reflective setAccessible(true) disabled
08:39:45.507 [main] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.Bits.unaligned: available, true
08:39:45.508 [main] DEBUG io.netty.util.internal.PlatformDependent0 - jdk.internal.misc.Unsafe.allocateUninitializedArray(int): unavailable: class io.netty.util.internal.PlatformDependent0$7 cannot access class jdk.internal.misc.Unsafe (in module java.base) because module java.base does not export jdk.internal.misc to unnamed module @c540f5a
08:39:45.597 [main] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.DirectByteBuffer.<init>(long, int): unavailable
08:39:45.597 [main] DEBUG io.netty.util.internal.PlatformDependent - sun.misc.Unsafe: available
08:39:45.598 [main] DEBUG io.netty.util.internal.PlatformDependent - maxDirectMemory: 127729664 bytes (maybe)
08:39:45.598 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.tmpdir: /tmp (java.io.tmpdir)
08:39:45.598 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.bitMode: 64 (sun.arch.data.model)
08:39:45.599 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.maxDirectMemory: -1 bytes
08:39:45.599 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.uninitializedArrayAllocationThreshold: -1
08:39:45.601 [main] DEBUG io.netty.util.internal.CleanerJava9 - java.nio.ByteBuffer.cleaner(): available
08:39:45.601 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.noPreferDirect: false
08:39:46.198 [main] DEBUG io.netty.channel.MultithreadEventLoopGroup - -Dio.netty.eventLoopThreads: 2
08:39:46.297 [main] DEBUG io.netty.util.internal.InternalThreadLocalMap - -Dio.netty.threadLocalMap.stringBuilder.initialSize: 1024
08:39:46.298 [main] DEBUG io.netty.util.internal.InternalThreadLocalMap - -Dio.netty.threadLocalMap.stringBuilder.maxSize: 4096
08:39:46.303 [main] DEBUG io.netty.channel.nio.NioEventLoop - -Dio.netty.noKeySetOptimization: false
08:39:46.304 [main] DEBUG io.netty.channel.nio.NioEventLoop - -Dio.netty.selectorAutoRebuildThreshold: 512
08:39:46.310 [main] DEBUG io.netty.util.internal.PlatformDependent - org.jctools-core.MpscChunkedArrayQueue: available
08:39:47.107 [grpc-default-executor-0] DEBUG io.netty.util.ResourceLeakDetector - -Dio.netty.leakDetection.level: simple
08:39:47.108 [grpc-default-executor-0] DEBUG io.netty.util.ResourceLeakDetector - -Dio.netty.leakDetection.targetRecords: 4
08:39:47.198 [grpc-default-executor-0] DEBUG io.netty.buffer.AbstractByteBuf - -Dio.netty.buffer.checkAccessible: true
08:39:47.198 [grpc-default-executor-0] DEBUG io.netty.buffer.AbstractByteBuf - -Dio.netty.buffer.checkBounds: true
08:39:47.200 [grpc-default-executor-0] DEBUG io.netty.util.ResourceLeakDetectorFactory - Loaded default ResourceLeakDetector: io.netty.util.ResourceLeakDetector@17cba4d9
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.numHeapArenas: 2
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.numDirectArenas: 2
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.pageSize: 8192
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxOrder: 9
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.chunkSize: 4194304
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.smallCacheSize: 256
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.normalCacheSize: 64
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxCachedBufferCapacity: 32768
08:39:47.404 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.cacheTrimInterval: 8192
08:39:47.405 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.cacheTrimIntervalMillis: 0
08:39:47.405 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.useCacheForAllThreads: false
08:39:47.405 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxCachedByteBuffersPerChunk: 1023
08:39:47.603 [grpc-default-executor-0] DEBUG io.netty.channel.DefaultChannelId - -Dio.netty.processId: 1 (auto-detected)
08:39:47.606 [grpc-default-executor-0] DEBUG io.netty.util.NetUtil - -Djava.net.preferIPv4Stack: false
08:39:47.606 [grpc-default-executor-0] DEBUG io.netty.util.NetUtil - -Djava.net.preferIPv6Addresses: false
08:39:47.608 [grpc-default-executor-0] DEBUG io.netty.util.NetUtilInitializations - Loopback interface: lo (lo, 127.0.0.1)
08:39:47.609 [grpc-default-executor-0] DEBUG io.netty.util.NetUtil - /proc/sys/net/core/somaxconn: 1024
08:39:47.610 [grpc-default-executor-0] DEBUG io.netty.channel.DefaultChannelId - -Dio.netty.machineId: 92:85:37:ff:fe:15:40:79 (auto-detected)
08:39:47.713 [grpc-default-executor-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.allocator.type: pooled
08:39:47.713 [grpc-default-executor-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.threadLocalDirectBufferSize: 0
08:39:47.713 [grpc-default-executor-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.maxThreadLocalCharBufferSize: 16384
08:39:47.801 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.maxCapacityPerThread: 4096
08:39:47.801 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.ratio: 8
08:39:47.801 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.chunkSize: 32
08:39:47.801 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.blocking: false
08:39:47.900 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND SETTINGS: ack=false settings={ENABLE_PUSH=0, MAX_CONCURRENT_STREAMS=0, INITIAL_WINDOW_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192}
08:39:47.911 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=983041
08:39:48.016 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND SETTINGS: ack=false settings={MAX_CONCURRENT_STREAMS=2147483647, INITIAL_WINDOW_SIZE=1048576, MAX_HEADER_LIST_SIZE=8192}
08:39:48.017 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND SETTINGS: ack=true
08:39:48.110 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND WINDOW_UPDATE: streamId=0 windowSizeIncrement=983041
08:39:48.197 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND SETTINGS: ack=true
08:39:48.298 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND HEADERS: streamId=3 headers=GrpcHttp2OutboundHeaders[:authority: zell-test-zeebe-gateway:26500, :path: /gateway_protocol.Gateway/Topology, :method: POST, :scheme: http, content-type: application/grpc, te: trailers, user-agent: zeebe-client-java/8.2.0-SNAPSHOT grpc-java-netty/1.53.0, grpc-accept-encoding: gzip, grpc-timeout: 60500737u] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false
08:39:48.398 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND DATA: streamId=3 padding=0 endStream=true length=5 bytes=0000000000
08:39:48.404 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND PING: ack=false bytes=1234
08:39:48.404 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND PING: ack=true bytes=1234
08:39:48.407 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND HEADERS: streamId=3 headers=GrpcHttp2ResponseHeaders[:status: 200, content-type: application/grpc, grpc-encoding: identity, grpc-accept-encoding: gzip] padding=0 endStream=false
08:39:48.500 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND DATA: streamId=3 padding=0 endStream=false length=113 bytes=000000006c0a54123e7a656c6c2d746573742d7a656562652d302e7a656c6c2d746573742d7a656562652e7a656c6c2d6d697865642d696e7665737469676174...
08:39:48.500 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND PING: ack=false bytes=1234
08:39:48.503 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND HEADERS: streamId=3 headers=GrpcHttp2ResponseHeaders[grpc-status: 0] padding=0 endStream=true
08:39:48.597 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND PING: ack=true bytes=1234
08:39:48.700 [main] INFO  io.camunda.zeebe.App - Broker 0 - zell-test-zeebe-0.zell-test-zeebe.zell-mixed-investigation.svc:26501
08:39:48.803 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND HEADERS: streamId=5 headers=GrpcHttp2OutboundHeaders[:authority: zell-test-zeebe-gateway:26500, :path: /gateway_protocol.Gateway/DeployResource, :method: POST, :scheme: http, content-type: application/grpc, te: trailers, user-agent: zeebe-client-java/8.2.0-SNAPSHOT grpc-java-netty/1.53.0, grpc-accept-encoding: gzip, grpc-timeout: 61999279u] streamDependency=0 weight=16 exclusive=false padding=0 endStream=false
08:39:48.804 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] OUTBOUND DATA: streamId=5 padding=0 endStream=true length=2557 bytes=00000009f80af5130a1262706d6e2f6f6e655f7461736b2e62706d6e12de133c3f786d6c2076657273696f6e3d22312e302220656e636f64696e673d22555446...
08:39:48.830 [grpc-nio-worker-ELG-1-1] DEBUG io.grpc.netty.NettyClientHandler - [id: 0xb44fde55, L:/10.0.12.70:35022 - R:zell-test-zeebe-gateway/10.4.7.140:26500] INBOUND HEADERS: streamId=5 headers=GrpcHttp2ResponseHeaders[:status: 200, content-type: application/grpc, grpc-status-details-bin: CA4Sb0V4cGVjdGVkIHRvIGV4ZWN1dGUgY29tbWFuZCBvbiBwYXJ0aXRpb24gMSwgYnV0IGVpdGhlciBpdCBkb2VzIG5vdCBleGlzdCwgb3IgdGhlIGdhdGV3YXkgaXMgbm90IHlldCBhd2FyZSBvZiBpdA, grpc-status: 14, grpc-message: Expected to execute command on partition 1, but either it does not exist, or the gateway is not yet aware of it] padding=0 endStream=true
```
Timer log, which prints the config, shows that it tries to connect to localhost.

```sh
[cqjawa zeebe/ cluster: zeebe-cluster ns:zell-mixed-investigation]$ k logs timer-77655fb49b-t7t97 
08:39:34.699 [main] INFO  io.camunda.zeebe.App - Starting app with config: {
    # application.conf @ file:/app/resources/application.conf: 2
    "brokerUrl" : "localhost:26500",
    # application.conf @ file:/app/resources/application.conf: 5
    "monitoringPort" : 9600,
    # application.conf @ file:/app/resources/application.conf: 7
    "starter" : {
        # application.conf @ file:/app/resources/application.conf: 11
        "bpmnXmlPath" : "bpmn/one_task.bpmn",
        # application.conf @ file:/app/resources/application.conf: 15
        "durationLimit" : 0,
        # application.conf @ file:/app/resources/application.conf: 16
        "msgName" : "msg",
        # application.conf @ file:/app/resources/application.conf: 12
        "payloadPath" : "bpmn/big_payload.json",
        # application.conf @ file:/app/resources/application.conf: 8
        "processId" : "benchmark",
        # application.conf @ file:/app/resources/application.conf: 9
        "rate" : 300,
        # application.conf @ file:/app/resources/application.conf: 17
        "startViaMessage" : false,
        # application.conf @ file:/app/resources/application.conf: 10
        "threads" : 2,
        # application.conf @ file:/app/resources/application.conf: 13
        "withResults" : false,
        # application.conf @ file:/app/resources/application.conf: 14
        "withResultsTimeout" : "60s"
    },
    # application.conf @ file:/app/resources/application.conf: 4
    "tls" : false,
    # application.conf @ file:/app/resources/application.conf: 20
    "worker" : {
        # application.conf @ file:/app/resources/application.conf: 24
        "capacity" : 30,
        # application.conf @ file:/app/resources/application.conf: 27
        "completeJobsAsync" : false,
        # application.conf @ file:/app/resources/application.conf: 26
        "completionDelay" : "300ms",
        # application.conf @ file:/app/resources/application.conf: 21
        "jobType" : "benchmark-task",
        # application.conf @ file:/app/resources/application.conf: 28
        "payloadPath" : "bpmn/big_payload.json",
        # application.conf @ file:/app/resources/application.conf: 25
        "pollingDelay" : "1s",
        # application.conf @ file:/app/resources/application.conf: 23
        "threads" : 10,
        # application.conf @ file:/app/resources/application.conf: 22
        "workerName" : "benchmark-worker"
    }
}

08:39:36.600 [main] DEBUG io.netty.util.internal.logging.InternalLoggerFactory - Using SLF4J as the default logging framework
08:39:36.604 [main] DEBUG io.netty.util.internal.PlatformDependent0 - -Dio.netty.noUnsafe: false
08:39:36.604 [main] DEBUG io.netty.util.internal.PlatformDependent0 - Java version: 17
08:39:36.605 [main] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.theUnsafe: available
08:39:36.606 [main] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.copyMemory: available
08:39:36.607 [main] DEBUG io.netty.util.internal.PlatformDependent0 - sun.misc.Unsafe.storeFence: available
08:39:36.607 [main] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.Buffer.address: available
08:39:36.607 [main] DEBUG io.netty.util.internal.PlatformDependent0 - direct buffer constructor: unavailable: Reflective setAccessible(true) disabled
08:39:36.697 [main] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.Bits.unaligned: available, true
08:39:36.698 [main] DEBUG io.netty.util.internal.PlatformDependent0 - jdk.internal.misc.Unsafe.allocateUninitializedArray(int): unavailable: class io.netty.util.internal.PlatformDependent0$7 cannot access class jdk.internal.misc.Unsafe (in module java.base) because module java.base does not export jdk.internal.misc to unnamed module @c540f5a
08:39:36.699 [main] DEBUG io.netty.util.internal.PlatformDependent0 - java.nio.DirectByteBuffer.<init>(long, int): unavailable
08:39:36.700 [main] DEBUG io.netty.util.internal.PlatformDependent - sun.misc.Unsafe: available
08:39:36.700 [main] DEBUG io.netty.util.internal.PlatformDependent - maxDirectMemory: 127729664 bytes (maybe)
08:39:36.700 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.tmpdir: /tmp (java.io.tmpdir)
08:39:36.701 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.bitMode: 64 (sun.arch.data.model)
08:39:36.701 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.maxDirectMemory: -1 bytes
08:39:36.702 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.uninitializedArrayAllocationThreshold: -1
08:39:36.703 [main] DEBUG io.netty.util.internal.CleanerJava9 - java.nio.ByteBuffer.cleaner(): available
08:39:36.703 [main] DEBUG io.netty.util.internal.PlatformDependent - -Dio.netty.noPreferDirect: false
08:39:37.297 [main] DEBUG io.netty.channel.MultithreadEventLoopGroup - -Dio.netty.eventLoopThreads: 2
08:39:37.401 [main] DEBUG io.netty.util.internal.InternalThreadLocalMap - -Dio.netty.threadLocalMap.stringBuilder.initialSize: 1024
08:39:37.401 [main] DEBUG io.netty.util.internal.InternalThreadLocalMap - -Dio.netty.threadLocalMap.stringBuilder.maxSize: 4096
08:39:37.406 [main] DEBUG io.netty.channel.nio.NioEventLoop - -Dio.netty.noKeySetOptimization: false
08:39:37.406 [main] DEBUG io.netty.channel.nio.NioEventLoop - -Dio.netty.selectorAutoRebuildThreshold: 512
08:39:37.413 [main] DEBUG io.netty.util.internal.PlatformDependent - org.jctools-core.MpscChunkedArrayQueue: available
08:39:38.600 [grpc-default-executor-0] DEBUG io.netty.util.ResourceLeakDetector - -Dio.netty.leakDetection.level: simple
08:39:38.600 [grpc-default-executor-0] DEBUG io.netty.util.ResourceLeakDetector - -Dio.netty.leakDetection.targetRecords: 4
08:39:38.607 [grpc-default-executor-0] DEBUG io.netty.buffer.AbstractByteBuf - -Dio.netty.buffer.checkAccessible: true
08:39:38.608 [grpc-default-executor-0] DEBUG io.netty.buffer.AbstractByteBuf - -Dio.netty.buffer.checkBounds: true
08:39:38.698 [grpc-default-executor-0] DEBUG io.netty.util.ResourceLeakDetectorFactory - Loaded default ResourceLeakDetector: io.netty.util.ResourceLeakDetector@2a83ac19
08:39:39.105 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.numHeapArenas: 2
08:39:39.105 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.numDirectArenas: 2
08:39:39.105 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.pageSize: 8192
08:39:39.105 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxOrder: 9
08:39:39.105 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.chunkSize: 4194304
08:39:39.105 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.smallCacheSize: 256
08:39:39.106 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.normalCacheSize: 64
08:39:39.106 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxCachedBufferCapacity: 32768
08:39:39.106 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.cacheTrimInterval: 8192
08:39:39.106 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.cacheTrimIntervalMillis: 0
08:39:39.106 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.useCacheForAllThreads: false
08:39:39.106 [grpc-default-executor-0] DEBUG io.netty.buffer.PooledByteBufAllocator - -Dio.netty.allocator.maxCachedByteBuffersPerChunk: 1023
08:39:39.206 [grpc-default-executor-0] DEBUG io.netty.channel.DefaultChannelId - -Dio.netty.processId: 1 (auto-detected)
08:39:39.208 [grpc-default-executor-0] DEBUG io.netty.util.NetUtil - -Djava.net.preferIPv4Stack: false
08:39:39.208 [grpc-default-executor-0] DEBUG io.netty.util.NetUtil - -Djava.net.preferIPv6Addresses: false
08:39:39.209 [grpc-default-executor-0] DEBUG io.netty.util.NetUtilInitializations - Loopback interface: lo (lo, 127.0.0.1)
08:39:39.210 [grpc-default-executor-0] DEBUG io.netty.util.NetUtil - /proc/sys/net/core/somaxconn: 1024
08:39:39.297 [grpc-default-executor-0] DEBUG io.netty.channel.DefaultChannelId - -Dio.netty.machineId: 7e:55:fb:ff:fe:0c:e8:56 (auto-detected)
08:39:39.312 [grpc-default-executor-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.allocator.type: pooled
08:39:39.312 [grpc-default-executor-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.threadLocalDirectBufferSize: 0
08:39:39.312 [grpc-default-executor-0] DEBUG io.netty.buffer.ByteBufUtil - -Dio.netty.maxThreadLocalCharBufferSize: 16384
08:39:39.400 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.maxCapacityPerThread: 4096
08:39:39.400 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.ratio: 8
08:39:39.400 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.chunkSize: 32
08:39:39.400 [grpc-default-executor-0] DEBUG io.netty.util.Recycler - -Dio.netty.recycler.blocking: fals

```

The starter application loops in the printTopology https://github.com/camunda/zeebe/blob/main/benchmarks/project/src/main/java/io/camunda/zeebe/App.java#L68 silently, since errors are not logged.

**Comment by Zelldon** [comment]
Fixed via zeebe-io/benchmark-helm#33

**Comment by Zelldon** [comment]
Ok upgraded all medic mixed benchmarks

![benchmarks](https://user-images.githubusercontent.com/2758593/222166108-bb8ab361-b802-4acb-a9e6-05a685f5cd8f.png)
